client/state/first-view/actions.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import {
+	FIRST_VIEW_EMBRACE,
+	FIRST_VIEW_DISMISS,
+	FIRST_VIEW_HIDE,
+	FIRST_VIEW_SHOW,
+} from 'state/action-types';
+
+import { FIRST_VIEW_START_DATES } from './constants';
+
+export function init( { user } ) {
+	const userCreated = moment( user.get().date );
+	let eligibleViews = [];
+
+	for ( let view in FIRST_VIEW_START_DATES ) {
+		if ( moment( FIRST_VIEW_START_DATES[ view ] ).isBefore( userCreated ) ) {
+			eligibleViews.push( view );
+		}
+	}
+
+	return {
+		type: FIRST_VIEW_SHOW,
+		views: eligibleViews,
+	};
+}
+
+export function dismissView( { view } ) {
+	return {
+		type: FIRST_VIEW_DISMISS,
+		view,
+	};
+}
+
+export function embraceView( { view } ) {
```

Comment:
Hmm... what do you think about `enableView` with a bool parameter? Or even `showViewNextTime`? `dismiss` is a bit ambiguous (I used that term for hiding the view with the 'Got it!' button). 

But, regardless of what you decide here, I'll make sure the UI code uses the same terminology! :smile: